Dido's project/CLKW208HoodRichBulletproofEdition.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CLK W208 HoodRich Bulletproof Edition</title>
+    <link rel="stylesheet" href="./StylesIMG2.css" />
+  </head>
+  <body>
+    <main>
+      <header class="Header">
+        <div class="Text1">
+          <h1>CLK W208 HoodRich Bulletproof Edition</h1>
+        </div>
+        <div class="CLK">
+          <img src="./img/CLK.jpg" height="400vh" />
+        </div>
+        <div>
+          <h2>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim,
+            temporibus!
+          </h2>
+          <div class="Button">
```

Comment:
Make sure that the name of the classes fits the purpose of the usage.
Maybe name this one `centered-container`